src/blockscoutApi/index.ts

```diff
@@ -127,12 +127,33 @@ export class BlockscoutAPI extends DataSource {
     address, topic0, toBlock = 'latest', fromBlock
   }: Omit<GetEventLogsByAddressAndTopic0, 'chainId'>) {
     let fromBlockToUse = fromBlock
+
     if (!fromBlock) {
-      const tx = await this.getTransactionsByAddress(address)
-      // @ts-ignore ignored because it's using never as type
-      const lastTx = tx.data.pop() // The last tx is the first transaction
+      try {
+        const params = {
+          module: 'account',
+          action: 'txlist',
+          address,
+          sort: 'asc'
+        }
+        const response = await this.axios?.get<BlockscoutTransactionResponse>(this.url, { params })
```

Comment:
We should reuse ServerResponse
```suggestion
        const response = await this.axios?.get<ServerResponse<BlockscoutTransactionResponseTxResult>>(this.url, { params })
```